lib/modules/apostrophe-pages/lib/api.js

```diff
@@ -690,8 +690,60 @@ module.exports = function(self, options) {
   //
   // See `moveToTrash` for what the callback receives.
 
-  self.trashInSchema = function(req, _id, callback) {
-    return self.trashInSchemaBody(req, _id, true, callback);
+  self.trashInSchema = function(req, _id, toTrash, callback) {
```

Comment:
I see you're restoring the old signature here; arguably changing in 2.58.0 was a bc break and we're just fixing that, so OK.